modules/op-release-notes-1-10.adoc

```diff
@@ -0,0 +1,175 @@
+=== {pac}
+
+* This update enhances incoming webhook support. For a GitHub application installed on the cluster, you do not need to provide the `git_provider` specification for an incoming webhook. Instead, {pac} detects the secret and use it for the incoming webhook.
+* With this update, you can use the same token to fetch remote tasks from the same host on GitHub with a non-default branch.
+* With this update, {pac} supports Tekton v1 templates. You can have v1 and v1beta1 templates, which {pac} reads for PR generation. The PR is created as v1 on cluster.
+* Before this update, OpenShift console UI would use a hardcoded pipeline run template as a fallback template when a runtime template was not found in the OpenShift namespace. This update provides a default pipeline run template for the console to use, `pipelines-as-code-template-default`, in the `pipelines-as-code` config map. 
+* With this update, {pac} supports Tekton Pipelines 0.44.0 minimal status. 
+* With this update, {pac} supports Tekton v1 API, which means {pac} is now compatible with Tekton v0.44 and later.
+* With this update, you can configure custom console dashboards in addition to configuring a console for OpenShift and Tekton dashboards for k8s.
+* With this update, {pac} detects the installation of a GitHub application initiated using the `tkn pac create repo` command and does not require a GitHub webhook if it was installed globally.
+* Before this update, if there was an error on a `PipelineRun` execution and not on the tasks attached to `PipelineRun`, {pac} would not report the failure properly. With this update, {pac} reports the error properly on the GitHub checks when a `PipelineRun` could not get created.  
+* With this update, {pac} includes a `target_namespace` variable, which expands to the currently running namespace where the `PipelineRun` is executed.
+* With this update, {pac} lets you bypass GitHub enterprise questions in the CLI bootstrap GitHub application.
+* With this update, {pac} does not report errors when the repository CR was not found.
+* With this update, {pac} reports an error if multiple pipeline runs with the same name were found.
+
+
+[id="breaking-changes-1-10_{context}"]
+== Breaking changes
+
+* With this update, the prior version of the `tkn` command is not compatible with {pipelines-title} 1.10.
+* This update removes support for cluster and `CloudEvent` pipeline resources from Tekton CLI. You cannot create pipeline resources by using the `tkn pipelineresource create` command. Also, the pipeline resources are not supported anymore in the `start` command of a task, cluster task, and pipeline.
+* This update removes `tekton` as a provenance format from Tekton Chains.
+
+
+[id="deprecated-features-1-10_{context}"]
+== Deprecated and removed features
+
+* In {pipelines-title} 1.10, the `ClusterTask` commands are now deprecated and are planned to be removed in a future release. The `tkn task create` command is also deprecated with this update. 
```

Comment:
@piyush-garg `tkn task create` deprecated ?

Reply:
@piyush-garg Could you please confirm the same?